### PR TITLE
Add new French translations for Stretchly redesign

### PR DESCRIPTION
**Pull request by whiver**
Add new French translations for Stretchly redesign.

Issue:  #584

### Requirements

- [x]  issue was opened to discuss proposed changes before starting implementation.
- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.

### Description of the Change

New French translations for the app redesign.

### Verification Process

Not verified yet.

### CODE REVIEW

File: app/locales/fr.json [hunk 1]
```diff
@@ -17,171 +17,171 @@
...
-    "aboutMinutesRemaining": "About $t(utils.minutes, {'count': {{minutes}} }) remaining",
-    "aboutHoursRemaining": "About $t(utils.hours, {'count': {{hours}} }) remaining",
-    "aboutHoursMinutesRemaining": "About $t(utils.hours, {'count': {{hours}} }) $t(utils.minutes, {'count': {{minutes}} }) remaining",
-    "inSeconds": "in $t(utils.seconds, {'count': {{seconds}} })",
-    "inAboutMinutes": "in about $t(utils.minutes, {'count': {{minutes}} })",
-    "inAboutHours": "in about $t(utils.hours, {'count': {{hours}} })",
-    "inAboutHoursMinutes": "in about $t(utils.hours, {'count': {{hours}} }) $t(utils.minutes, {'count': {{minutes}} })"
+    "hours": "{{count}} heure",
+    "hours_plural": "{{count}} heures",
+    "secondsRemaining": "$t(utils.seconds, {'count': {{seconds}} }) restantes",
```
Comment (whiver) [inline app/locales/fr.json:158]: @hovancik We could use a plural version of this one and the variants below too.

Comment (hovancik) [reply app/locales/fr.json:158]: do you need plural in French there?

In English, also in Slovak it's the same word. 

`secondsRemaining` is 1-60 seconds
`aboutMinutesRemaining` is 2-60 minutes
`aboutHoursRemaining` is 2 - xxx hours
`aboutHoursMinutesRemaining` is 2+hours 1-59 minutes

Comment (whiver) [reply app/locales/fr.json:158]: The original string was `"$t(utils.seconds, {'count': {{seconds}} }) remaining"`. In this string, "remaining" has a plural form in French:
- 1 seconde restante
- 2 secondes restante**s**

So yes, a plural form here would be helpful :slightly_smiling_face:

Comment (hovancik) [reply app/locales/fr.json:158]: I will have to think, because I don't see way to do it right now. (With my nested form)

Comment (whiver) [reply app/locales/fr.json:158]: Well that's not a big deal, we can still sort this out later :)

Comment (hovancik) [reply app/locales/fr.json:158]: Ok :+1: I'll have to check if some libraries exist for this

**Comment by hovancik** [comment]
@whiver I can merge this, then?

**Comment by whiver** [comment]
Sure, you can! Thanks :)

**Pull request merged**